src/olympia/addons/admin.py

```diff
@@ -177,8 +180,9 @@ class Media:
         }))
     actions = ['git_extract_action']
 
-    def queryset(self, request):
-        return models.Addon.unfiltered
+    def get_queryset(self, request):
+        return Addon.unfiltered.all().only_translations().transform(
+            Addon.attach_all_authors)
```

Comment:
This is going to change the queryset for the `change_view` too and we don't need to attach_all_authors there (... right??) 

I'm assuming it's not a terrible thing anyway, including bypassing the usual Addon transforms, as the tests pass and the query count seems to have gone down for the `change_view`.

Reply:
Yeah. Ideally we'd either be able to change the queryset for a given view or the inline for the authors would be able to use `all_authors` (or we'd be able to use `prefetch_related()`). But I don't think you can do either of these things in a clean/satisfying/safe way, so I figured I would live it like that for now.